_episodes/01-introduction.md

```diff
@@ -44,17 +44,15 @@ This code produces the following figure:
 > ## Notice
 > If you look at the plot and the order of the list of numbers you can clearly see that the order of the arguments is of the form
 >
-> ~~~
+> ```python
 > ax.plot(xpoints,ypoints)
-> ~~~
-> {: .language-python}
+> ```
 >
 >But what is useful is that if we wanted to do more than one plot in the same figure we could do this in two main ways.
-> ~~~
+```python
```

Comment:
Aren't you missing a `>` here?

Reply:
Generally probably good to test the local preview with these PRs

Reply:
it seems the github "Files Changed" wraps the statement?

Reply:
oh yeah sorry I thought u were referring to the line above. 
yes the three ticks were missing, I have update that

DONE